src/summaries/control_flow.yaml

```diff
@@ -0,0 +1,739 @@
+- title: Control Flow
+  shortTitle: Control flow
+  key: control_flow
+  summary: Many programming languages allow you to run some code depending on whether a condition is true, or run some code repeatedly while a condition is true. If expressions and loops are the most common constructs in Rust that control execution flow.
+  details: |
+    - In most cases, the computer runs the code in order from the first line in the file to the last line, unless it encounters structures that alter the control flow, such as loops and conditionals.
+    - We control the flow of our program with the following statements, in combination with relational comparison and logical operators.
+      * If else
+      * Match
+      * Loop
+      * For loop
+- title: If Expressions
+  shortTitle: If-else statement
+  key: if_else_let_statement
+  summary: We use if statement to give conditions to our code. You can branch your code based on conditions using an if expression. A condition is given and then a block of code is run if the condition is met. In the event that the condition is not met, the compiler says to the machine “do not run this block."
+  details: |
+    - To explore the if expression, create a new project called branches in your projects directory. Add the following code to src/main.rs:
```

Comment:
This seems to be a copy paste `To explore the if expression, create a new project called branches in your projects directory. Add the following code to src/main.rs:`

Reply:
edited and added more detail